MapboxNavigation/FeedbackViewController.swift

```diff
@@ -48,20 +63,27 @@ class FeedbackViewController: UIViewController, DismissDraggable, UIGestureRecog
     }
     
     class func loadFromStoryboard() -> FeedbackViewController {
-        let storyboard = UIStoryboard(name: "Navigation", bundle: .mapboxNavigation)
-        return storyboard.instantiateViewController(withIdentifier: "FeedbackViewController") as! FeedbackViewController
+        let feedbackVC = FeedbackViewController()
+        return feedbackVC
+//        let storyboard = UIStoryboard(name: "Navigation", bundle: .mapboxNavigation)
+//        return storyboard.instantiateViewController(withIdentifier: "FeedbackViewController") as! FeedbackViewController
```

Comment:
We should probably delete these 2 commented lines.

Reply:
Heh, that is a hack, the entire function will be going away.

Note that this is still a WIP, and as-such this code is still kinda crufty.